tomipaul/PostIt
# #151285073 Resolve issues in user interface/user experience

#### What does this PR do?
It adds logic to display preloader during image upload
#### Description of Task to be completed?
- Add an action that is dispatched during image upload
- Add a reducer that handles action and sets state to display preloader
#### How should this be manually tested?
- Navigate to https://app-postit.herokuapp.com
- Signup or login to application
- Click on the profile picture at the top of the sidebar to upload a new image
- A preloader should be shown during image upload
#### Any background context you want to provide?
Before there was no indication that a picture is being uploaded
Now a preloader is shown during image upload
#### What are the relevant pivotal tracker stories?
151285073 Resolve issues in user interface/user experience
